sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/rx/MultiOrderByQueryTests.java

```diff
@@ -7,6 +7,7 @@
 import com.azure.cosmos.CosmosAsyncClient;
```

Comment:
nit: Unused imports. (not related to this PR)

Reply:
Fixed